pandas/tests/scalar/timedelta/test_arithmetic.py

```diff
@@ -384,8 +384,11 @@ def test_td_div_nan(self, nan):
         result = td / nan
         assert result is NaT
 
-        result = td // nan
-        assert result is NaT
+        # TODO: Don't leave commented, this is just a temporary fix for
```

Comment:
make sure u open an issue

an xfail with a version test is the right way to handle this

i guess ok for right now

Reply:
@jreback there is already an issue (https://github.com/pandas-dev/pandas/issues/31992)

I also think that xfail is the right way to do this, but I am not sure about the value of the "reason" parameter.

Reply:
@MomIsBestFriend the issue will be to revert this PR, which is different. Since I assume that another PR will follow this immediately for that purpose, the issue may not be necessary.